src/features/finance/MealSlice.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-param-reassign */
```

Comment:
As mentioned above and as a best practice kindly avoid disabling linter checks in line.